pkg/install/stack/types.go

```diff
@@ -52,6 +52,11 @@ type DeployOptions struct {
 	EnableNginxIngressMigration        bool
 	DisableTelemetry                   bool
 	DisableDependencyUpdate            bool
+
```

Comment:
Can you please also adjust the mla-e2e job to actually use the installer to install MLA stuff?

Reply:
Yes, I wanted to do it in a separate PR just so I don't make this one much bigger. Is that OK?